piccolo/apps/playground/commands/run.py

```diff
@@ -326,7 +328,17 @@ def run(
                 "database": database,
                 "user": user,
                 "password": password,
-                "port": port,
+                "port": port or 5432,
+            }
+        )
+    elif engine.upper() == "COCKROACH":
+        db = CockroachEngine(
+            {
+                "host": host,
+                "database": database,
+                "user": "root",
+                "password": "",
```

Comment:
We have a `password` param too, so we might need to do the same as with `port` - make it optional, and if not specified then default to an empty string?

Or in the docs tell them to create the database with the same password as Postgres (`piccolo`).